bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -123,6 +123,12 @@ NB: Allowed ports for webhooks are 80, 88, 443 and 9443.
 
 ### Configure Things
 
+The easiest way to retrieve the IDs for all the devices and modules is to use the console command `openhab:netatmo ids`.
```

Comment:
Sorry, forgot one comment: Perhaps it would be better to rename the command to `showIds`? `ids` alone just mentions a noun without any action related to it.

Reply:
Done (good suggestion)